#goal definition
string formation_type
float64 position_x
float64 position_y
float64 heading
---
#result definition
float64[] placement_locations
---
#feedback
string feedback
